devito/operator.py

```diff
@@ -82,16 +82,17 @@ def __init__(self, nt, shape, dtype=np.float32, stencils=[],
             dimensions += [i for i in retrieve_dimensions(eqn.rhs) if i not in dimensions]
 
         # Time dimension is fixed for now
-        time_dim = t
+        time_dim = [t, time]
```

Comment:
You wouldn't from the diff, since it's already been added a while ago in `dimension.py`. The core idea is that `time` is unrolled and `t` is indexed, but that will be explained in accompanying documentation. It's also now aligned with the `save` flag on the `TimeData` constructor, so the user does not actually need to know this.